integration_tests/src/main/python/spark_init_internal.py

```diff
@@ -41,3 +41,5 @@ def get_spark_i_know_what_i_am_doing():
     """
     return _spark
 
+def spark_version():
```

Comment:
I like this, but there are a number of other places in the code that are doing a 

```
with_spark_session(spark: spark.sparkContext.version ...
```

It would probably be good to update all of them to be consistent or do it the same way here too.

Reply:
I have reverted the change to be more like the rest to be consistent